libparsec/crates/serialization_format/README.md

```diff
@@ -0,0 +1,39 @@
+# Libparsec Serialization Format
+
+This create provides function-like [`procedural macros`] for generating
```

Comment:
crate ?

Reply:
Fixed in https://github.com/Scille/parsec-cloud/pull/5080